ansible/roles/accumulo/templates/accumulo-cluster-systemd

```diff
@@ -287,26 +292,35 @@ function stop_all() {
   done
   wait
 
+{% if accumulo_version is version('2.1.0','<') %}
   for tracer in $(grep -E -v '(^#|^\s*$)' "${conf}/tracers"); do
     stop_service "$tracer" tracer &
   done
   wait
+{% endif %}
 
   # stop tserver still running
   stop_tservers
 
   echo "Cleaning all server entries in ZooKeeper"
+{% if accumulo_version is version('2.1.0','<') %}
   ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -master -tservers -tracers
+{% else %}
+  ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -manager -tservers
+{% endif %}
 }
 
 function stop_here() {
   # Determine hostname without errors to user
   hosts_to_check=("$(hostname -a 2> /dev/null | head -1)" "$(hostname -f)")
 
   for host in "${hosts_to_check[@]}"; do
-    for svc in  tserver gc master monitor tracer; do
+    for svc in  tserver gc master monitor; do
       stop_service "$host" "$svc"
     done
+{% if accumulo_version is version('2.1.0','<') %}
+    stop_service "$host" "tracer"
+{% endif %}
```

Comment:
This could be kept together too. It's probably just preference in the end over what looks better to you.
```suggestion
    for svc in  tserver gc master monitor{{ ' tracer' if accumulo_version is version('2.1.0','<') }}; do
      stop_service "$host" "$svc"
    done
```